### PR TITLE
#[161211657] As an admin user i should be able to add a product 

### PR DESCRIPTION
**Pull request by Ruiru11**
#### What does this PR do?
Have create and get endpoints  in products  for the store-API
#### Description of Task to be completed?
Have the following endpoint working:
> POST/api/v1/products
#### payload
```
{
       "name":"steel",
       "price":"sh.825",
       "category":"construction",
       "description":"Y16-For Columns and Slab"
}
```

>GET/api/v1/products
>GET/api/v1/products/<int:id>
#### How should this be manually tested?
After cloning the repo locally run python run.py
. Using postman test the above endpoints using the below header
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/161211657

### CODE REVIEW

**Comment by ghost** [comment]
Your PR title doesn't follow the specified naming conventions. Remove `finishes` from the title. Also, you don't have to use the brackets.

**Comment by ghost** [comment]
Specify the payload for the POST request.

e.g. POST [/api/v1/products](/api/v1/products)

payload
```
{
 name: "namehere",
 description: "description is here."
 price: 42
}
```

File: v1/app/views/products_views.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+from app.controllers.products_controllers import Products
+
+don_item = Blueprint('products', __name__, url_prefix="/api/v1")
+
+product_instance = Products()
+
+
+@don_item.route("/products", methods=["POST"])
+def create_item():
+    parser = reqparse.RequestParser()
```
Comment (ghost) [inline v1/app/views/products_views.py:32]: Use docstrings to indicate what's going on in your methods. Make it uniform across your code base.

**Comment by Ruiru11** [comment]
i have done some changes is that okay?

**Pull request merged**